best_practices/creating-the-project.rst

```diff
@@ -8,35 +8,12 @@ In the past, Symfony projects were created with `Composer`_, the dependency mana
 for PHP applications. However, the current recommendation is to use the **Symfony
 Installer**, which has to be installed before creating your first project.
 
-Linux and Mac OS X Systems
```

Comment:
Sorry for missing your comment. I've tried to add the labels in c8b16cf. Please tell me if it's OK. Thanks.